Sprint9-JS-Oject/tema3.js

```diff
@@ -0,0 +1,32 @@
+// Creați o șir de obiecte în care fiecare obiect descrie o carte și are proprietăți precum titlu (un șir de caractere - string), autor (un șir de caractere - string) și esteCitită (un boolean care indică dacă ați citit-o).
+
+const books = [
+    {
+        titlu: "Hobbitul",
+        autor: "J.R.R. Tolkien",
+        esteCitita: true,
+    },
+    {
+        titlu: "Harry Potter",
+        autor: "J.K. Rowling",
+        esteCitita: false,
+    }, 
+    {
+        titlu: "Atomic Habits",
+        autor: "James Clear",
+        esteCitita: false,
+    }
+];
+
+function getBooks(books) {
+    for (let book of books) {
+        if(book.esteCitita) {
```

Comment:
```suggestion
        if (book.esteCitita) {
```
Optional, recomand sa lasi cate un spatiu dupa cuvantul "if".
in general, poti folosi optiunea de "format"  din Visual Studio, ca sa-ti aranjeze el automat codul. In mare parte tu scrii bine, cu identarile potrivite, dar o sa vezi mici ajustari care intr-un final ajuta la citirea codului mai bine. De asemenea - te vei obisnui astfel sa scrii cod frumos din prima.

Reply:
done